doc/history.md

```diff
@@ -1,682 +1,903 @@
+# History
+
 ## 3.8.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.7.3
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.7.2
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.7.1
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.7.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.6.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.5.1
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.5.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.4.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.3.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.2.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.1.1
-* update mcdata
+
+- update `minecraft-data`
 
 ## 3.0.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.221.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.220.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.119.1
-* Alias versions that end in .0 to a version without the ending .0
+
+- Alias versions that end in .0 to a version without the ending .0
 
 ## 2.119.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.118.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.117.1
-* Add support for custom supportFeature return type
+
+- Add support for custom supportFeature return type
 
 ## 2.117.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.116.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.115.2
-* Add supportFeature support
+
+- Add supportFeature support
 
 ## 2.115.1
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.114.1
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.114.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.113.3
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.113.2
-* update mcdata
 
-## 2.113.1
-* update mcdata
+- update `minecraft-data`
 
 ## 2.113.1
-* Add block state IDs to all versions with block data
+
+- update `minecraft-data`
+- Add block state IDs to all versions with block data
 
 ## 2.113.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.112.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.111.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.110.0
-* update mcdata
+
+- update `minecraft-data`
 
 ## 2.109.0
-* update mcdata
```

Comment:
This breaks the consistency with other PrismarineJs package and i don't see why make that change .

Reply:
alright will do 👍